modules/core/include/opencv2/core/mat.hpp

```diff
@@ -3597,6 +3597,10 @@ CV_EXPORTS MatExpr operator + (const Mat& m, const MatExpr& e);
 CV_EXPORTS MatExpr operator + (const MatExpr& e, const Scalar& s);
 CV_EXPORTS MatExpr operator + (const Scalar& s, const MatExpr& e);
 CV_EXPORTS MatExpr operator + (const MatExpr& e1, const MatExpr& e2);
+template<typename _Tp, int m, int n> inline
+CV_EXPORTS MatExpr operator + (const Mat& a, const Matx<_Tp, m, n>& b) { return a + Mat(b); }
```

Comment:
> CV_EXPORTS

Remove for inlined functions.

> inline

Use "static inline" instead to avoid adding of new symbols into binaries.

Reply:
fixed